src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -88,12 +88,22 @@ public static bool IsNullableType(this TypeSymbol type)
         }
 
         public static TypeSymbol GetNullableUnderlyingType(this TypeSymbol type)
+        {
+            return type.GetNullableUnderlyingTypeWithAnnotations().TypeSymbol;
+        }
+
+        public static TypeSymbolWithAnnotations GetNullableUnderlyingType(this TypeSymbolWithAnnotations type)
```

Comment:
>public static TypeSymbolWithAnnotations GetNullableUnderlyingType(this TypeSymbolWithAnnotations type) [](start = 8, length = 102)

Consider moving this helper to TypeSymbolWithAnnotations, this is where other similar helpers are and this is not a TypeSymbol as the containing type implies